Containerfile.archlinux

```diff
@@ -0,0 +1,64 @@
+# Thanks to [GuestSneezeOSDev](https://github.com/GuestSneezeOSDev) for helping create this!
```

Comment:
Thank you @GuestSneezeOSDev for making this Arch Linux Containerfile in the first place! I'm giving you credit where credit is due. :-) I finally got it fully working and I'll merge it in soon.

Reply:
Thank you